src/lib/recipes.ts

```diff
@@ -422,6 +422,104 @@ const xiphosRecipes: IRecipe[] = [
   ),
 ];
 
+// HH-a "Javelin" half-truck recipes
+const javelinRecipes: IRecipe[] = [
+  // Garage production: 20 x Refined Materials → 1 x HH-a "Javelin"
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 20 }],
+    [{ stuff: Vehicles.Javelin, count: 1 }]
+  ),
+  // Mass Production Factory: 150 x Refined Materials → 3 crates of 3 x HH-a "Javelin" (9 total)
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 150 }],
+    [{ stuff: Vehicles.Javelin, count: 9 }]
+  ),
+];
+
+// HH-b "Hoplite" half-truck recipes
+const hopliteRecipes: IRecipe[] = [
+  // Garage production: 22 x Refined Materials → 1 x HH-b "Hoplite"
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 22 }],
+    [{ stuff: Vehicles.Hoplite, count: 1 }]
+  ),
+  // Mass Production Factory: 165 x Refined Materials → 3 crates of 3 x HH-b "Hoplite" (9 total)
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 165 }],
+    [{ stuff: Vehicles.Hoplite, count: 9 }]
+  ),
+];
+
+// HH-d "Peltast" half-truck recipes
+const peltastRecipes: IRecipe[] = [
+  // Garage production: 24 x Refined Materials → 1 x HH-d "Peltast"
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 24 }],
+    [{ stuff: Vehicles.Peltast, count: 1 }]
+  ),
+  // Mass Production Factory: 180 x Refined Materials → 3 crates of 3 x HH-d "Peltast" (9 total)
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 180 }],
+    [{ stuff: Vehicles.Peltast, count: 9 }]
+  ),
+];
+
+// Niska Mk. I Gun Motor Carriage recipes
+const niskaMkIRecipes: IRecipe[] = [
```

Comment:
Recipes:


Garage | 60 x Refined Materials | Niska Mk. I Gun Motor Carriage | 60  hits

Mass Production Factory | 432 x Refined Materials | 3 crates  of 3 x Niska Mk. I Gun Motor Carriage | 02:30:00
Mass Production Factory | 540 x Refined Materials | 4 crates  of 3 x Niska Mk. I Gun Motor Carriage | 03:20:00
Mass Production Factory | 630 x Refined Materials | 5 crates  of 3 x Niska Mk. I Gun Motor Carriage